packages/api/src/idpuser/controller.test.ts

```diff
@@ -0,0 +1,117 @@
+/** @format */
```

Comment:
Nothing in `stela` should have the Prettier format pragma, because the whole thing is Prettier-formatted.